src/sentry/static/sentry/app/components/projectHeader/projectSelector.jsx

```diff
@@ -1,21 +1,18 @@
+import {browserHistory} from 'react-router';
```

Comment:
Sorry my auto import plugin sorts imports alphabetically + groups them

Reply:
don't be sorry for this.